plugins/extractors/bigquery/README.md

```diff
@@ -60,17 +60,36 @@ source:
+| `resource.service`             | `bigquery`                                                                                                                                                                                                                                          |
+| `description`                  | `table description`                                                                                                                                                                                                                                 |
+| `profile.total_rows`           | `2100`                                                                                                                                                                                                                                              |
+| `profile.usage_count`          | `15`                                                                                                                                                                                                                                                |
+| `profile.joins`                | [][Join](#Join)                                                                                                                                                                                                                                     |
+| `profile.filters`              | [`"WHERE t.param_3 = 'the_param' AND t.column_1 = \"xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx\""`,`"WHERE event_timestamp >= TIMESTAMP(\"2021-10-29\", \"UTC\") AND event_timestamp < TIMESTAMP(\"2021-11-22T02:01:06Z\")"`]                                  |
+| `schema`                       | [][Column](#column)                                                                                                                                                                                                                                 |
+| `properties.partition_field`   | `created_at` - returns the time partitioning field md.TimePartitioning.Field                                                                                                                                                                        |
+| `properties.partition_data`    | `map[string]interface{}` - partition related data for time and range partitioning. Sample: "partition_data": {"partition_field": "data_date", "require_partition_filter": false, "time_partition": {"partition_by": "DAY","partition_expire": 0 } } |
+| `properties.clustering_fields` | `[]interface{}` - list of fields on which the table is clustered                                                                                                                                                                                    |
```

Comment:
`[]interface{}` is internal representation for the code. It would actually have a list of strings, CMIIW.

Reply:
Fixed